### DISCUSSION TITLE
Discussion for Project

### DISCUSSION THREAD

**Discussion by Surrealsteel** [TagStudioDev/TagStudio#137] [Ideas]
This may be pre-mature but I wanted to ask if its possible to open / enable a discussion for this project. 
I do not have any programming skills but I'm also curious if there is a way I can help or contribute in some other way. This idea is really interesting to me and I'd like to help if I can.

**Comment by Loran425** [comment]
You're always welcome to drop in here for ideas and feature requests. 
If you're looking for a more conversational setting, @CyanVoxel dropped a discord link in the pinned comment of the TagStudio video.

**Comment by xarvex** [comment]
GitHub allows for a place to have discussions rather than issues on the repository, which might be benefial to those who either don't want to use Discord or simply wish to have those conversations centered around here. That is what they referencing by enabling discussions on the project.

**Comment by Loran425** [comment]
Apparently I have not used github enough, I did not realize discussions were a feature of the platform

**Comment by tadpole256** [comment]
I actually came to the discussions for a reason similar to OP, I wanted to know if and how I can help. I just watched your YouTube video about Tag Studio and it is such an exciting idea!

**Comment by DaneJessen** [comment]
It really is such a cool project. The concept aligns with how my brain actually works, and is one of those "why isn't this a much bigger thing?" I am SUPER glad this is open source and being led by cool and interesting people and not being sat on by MS.
I would love to help as well and will check out the discord.
